sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -1352,27 +1320,22 @@ class MultiFileCloudParquetPartitionReader(
         if (fileBlockMeta.blocks.length == 0) {
```

Comment:
it does looks like the filePath might not need to be passed back in the ParquetFileInfoWithBlockMeta from filterBlocks here since I think everywhere we use it file is accessible so can just use that.   partValues might be similar so we should check. I think something I missed when adding the other multi-file support.

Reply:
we may need them (except partValues) to set input file blocks

``` scala
          InputFileUtils.setInputFileBlock(
            fileBufsAndMeta.partitionedFile.filePath,
            fileBufsAndMeta.partitionedFile.start,
            fileBufsAndMeta.partitionedFile.length)
```

Reply:
that is from the HostMemoryBuffersWithMeta right?  I'm talking about ParquetFileInfoWithBlockMeta. I think the filePath in there is only used in places it has direct access to the PartitionedFile, it does look like perhaps the only thing it gets is conversion from String to Path.  Its not a big deal.

Reply:
Oh, Sorry, I misunderstood. I didn't change anything about ParquetFileInfoWithBlockMeta. and I just checked the usage of filePath, and seems it is necessary for Coalescing reading. Let's just keep it.